guide/src/parallelism.md

```diff
@@ -1,66 +1,75 @@
 # Parallelism
 
-CPython has the infamous GIL (Global Interpreter Lock), which prevents developers
-from getting true parallelism when running pure Python code. While PyO3 needs to
-hold the GIL by default when called from Python, in order to allow manipulation
-of Python objects, you can release the GIL when executing Rust-only code to
-achieve true parallelism.
-
-The [`Python::allow_threads`] method temporarily releases the GIL, thus allowing other Python threads to run.
-
-```rust,ignore
-impl Python {
-    pub fn allow_threads<T, F>(self, f: F) -> T where F: Send + FnOnce() -> T {}
+CPython has the infamous [Global Interpreter Lock](https://docs.python.org/3/glossary.html#term-global-interpreter-lock), which prevents that several threads can execute Python code in parallel. This makes threading in Python a bad fit for [CPU-bound](https://stackoverflow.com/questions/868568/) tasks and often forces developers to accept the overhead of multiprocessing.
```

Comment:
It's not really grammatical to say "prevents that", "prevents" usually takes the gerund: "CPython has the infamous Global Interpreter Lock which prevents several threads from executing Python code in parallel."

I also think maybe it should be "Python bytecode" instead of "Python code", like in the glossary entry. Possibly the meaning is the same, but out of context and without thinking too much about it "executing Python code" would, to me, include operations during which the GIL is *not* held, like importing a module or opening a file or something.